moto/secretsmanager/models.py

```diff
@@ -87,7 +88,7 @@ def get_secret_value(self, secret_id, version_id, version_stage):
 
         if 'secret_string' not in secret_version and 'secret_binary' not in secret_version:
             raise ResourceNotFoundException(
```

Comment:
This should just raise a `SecretNotFoundException()`.

Reply:
No it shouldn't it gives a different message, as I've demonstrated above. This was the whole reason I made this PR.

Reply:
Oh I'm sorry... I mis read.

In this case........... Can you make _another_ class similar to the above :)?  I mostly want to keep the special unicode exceptions in 1 place.

Reply:
Out of interest, why use specialised classes rather than a variable passed to the same class? Wouldn't it be better to more directly mirror the AWS interface? The other exceptions for `secretsmanager` take the messages as arguments.

Reply:
As far as the AWS interface goes... how that's implemented on AWS's end is unknown to us. Moto will still return the proper data back to the client.

From moto code maintainability perspective, we would prefer to have very specific exceptions so it's easy to find where in the code base there are issues. In this case, because there are odd things going on with unicode characters, I would like to keep that centralized as much as possible and not have odd permutations of it.

In most cases, where users are passing in custom exception messages, I typically don't care and will often prefer it for code reuse. However, because of the oddities with the unicode characters, I would like to have this centralized and isolated as much as possible.

Reply:
Got it! Changes made.